src/index.ts

```diff
@@ -140,7 +152,10 @@ joplin.plugins.register({
             execute: async () => {
                 const template = await getTemplateFromId(await joplin.settings.value("defaultTodoTemplateId"));
                 if (template) {
-                    await joplin.commands.execute("newTodo", await parser.parseTemplate(template.body));
+                    const parsedTemplate = await parser.parseTemplate(template.body);
+                    if (parsedTemplate) {
+                        await joplin.commands.execute("newTodo", parsedTemplate);
+                    }
```

Comment:
I said this before, but when you have identical code duplicated across 5 different places in the codebase, it's probably time to move it in to a function.

Reply:
Yup, got it. Actually, at that time there were only 2 places of repetition, but now this definitely can be refactored. Thanks for pointing it out.

Reply:
done